cli-setup.sh

```diff
@@ -143,14 +143,18 @@ parse_args() {
 
 init_venv_python() {
     debugging "Virtualenv: ${venv} doesn't not exist, Configuring."
-    for ver in {3,2,''}; do #ensure python3 is first
-	debugging "Checking Python${ver}: $(which python${ver})"
-        if (which python${ver} > /dev/null 2>&1); then
-	    python_version="$(python${ver} <<< 'import sys; print(sys.version_info[0])')"
-	    stdout_log "Python Version Selected: python${python_version}"
-	    break
-        fi
-    done
+    if [[ ${platform} == 'centos' ]]; then
```

Comment:
I thought `Centos/RHEL 8` used `Python 3` by default. Would this cause issues for CentOS 8 setups?
https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/#:~:text=In%20RHEL%208%2C%20Python%203.6,generally%20have%20the%20python3%20prefix.

Reply:
We are not supporting CentOS 8 anyway at the moment.